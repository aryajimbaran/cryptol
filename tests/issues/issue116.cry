t = (0x1, { key = 4, value = "goodbye"})

bug116Fixed = (t.1.value == t.1.value)
