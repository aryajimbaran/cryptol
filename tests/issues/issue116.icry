:l issue116.cry
sanityCheck
bug116fixed
