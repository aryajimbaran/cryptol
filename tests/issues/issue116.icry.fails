Known problem, see ticket #116.
