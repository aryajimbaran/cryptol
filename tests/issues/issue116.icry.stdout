Should be

True
True 

(and warnings) instead of a bunch of constraint errors
